system-tests/services/test-external-volumes.js

```diff
@@ -0,0 +1,62 @@
+require("../_support/utils/ServicesUtil");
+
+describe("Services", function() {
+  /**
+   * Test the pods
+   */
+  describe("External Volumes", function() {
+    afterEach(() => {
+      cy.window().then(win => {
+        win.location.href = "about:blank";
+      });
+    });
+
+    it("should be persistent after suspension in single container services", function() {
+      cy.visitUrl(
```

Comment:
I'm not convinced we need a separate test suite for external volumes here. I feel a more general test suite for just volumes would be better.